### DISCUSSION TITLE
Option to turn spectrum-Toast off?

### DISCUSSION THREAD

**Discussion by cfepdan** [Budibase/budibase#5083] [Ideas]
Is there, or could there be an option to turn spectrum-Toasts off? These little guys show me "Row saved" all the time, and I don't need `em.

I think I can add an embed block and wipe them out with jquery, but that's... inelegant.

**Comment by shogunpurple** [comment]
Hi @cfepdan - we currently don't have that option, but I'd be wondering why you would like to turn them off. Do you want to use a custom one? Or do you just not want any feedback in the UI at all when you save?

Thanks!

**Reply by cfepdan** [reply]
@shogunpurple we have a long multi-step form for which we update the row after every step, because there are times when the form will not reach completion but it's still important to get as much data as we've collected. It's not relevant--and distracting--to our end user to be notified that the row has been saved. I'd like the option to turn them on in a debug mode and off in production.

**Comment by cfepdan** [comment]
For now, unless there's an easier way to add this CSS, I'm adding an embed to my layout:

```
<img style="display:none;" src="0" onerror = "javascript:{   
var style = document.createElement('style');
style.appendChild(document.createTextNode('.spectrum-Toast--success { display: none !important;}'));
document.head.appendChild(style);
}">
```
